styles.css

```diff
@@ -128,23 +175,1194 @@ ul {
   display: flex;
   width: 327px;
   height: 164px;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
+  font-weight: 400px;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
   color: #172b4d;
   flex: none;
   order: 1;
   flex-grow: 0;
 }
 
 .social_icon_container {
-  display: flex;
-  justify-content: center;
+  width: 170.17px;
+  height: 20px;
+  flex: none;
   order: 2;
   flex-grow: 0;
 }
 
-.social-icon {
+.social_icon_container ul {
+  position: absolute;
+  width: 170.17px;
+  height: 20px;
+  justify-content: space-between;
+}
+
+.twitter {
+  /* position: absolute; */
+  left: 27.31%;
+  right: 67.36%;
+  top: 73.83%;
+  bottom: 23.42%;
+  background: #fff;
+}
+
+.github {
+  /* position: absolute; */
+  left: 59.04%;
+  right: 36.16%;
+  top: 73.83%;
+  bottom: 23.42%;
+  background: #fff;
+}
+
+.linkedin {
+  /* position: absolute; */
+  left: 37.71%;
+  right: 56.96%;
+  top: 73.83%;
+  bottom: 23.42%;
+  background: #fff;
+}
+
+.medium {
+  /* position: absolute; */
+  left: 48.38%;
+  right: 46.29%;
+  top: 74.1%;
+  bottom: 23.69%;
+  background: #fff;
+}
+
+.angellist {
+  /* position: absolute; */
+  left: 68.91%;
+  right: 27.31%;
+  top: 73.83%;
+  bottom: 23.42%;
+  background: #fff;
+}
+
+/* .social-icon {
   flex-direction: column;
   fill: #505f79;
   padding-left: 1.6em;
   width: 2em;
 }
+.scroll {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 8px;
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    left: calc(50% - 40px/2 + 0.5px);
+    bottom: 4px;
+    border: 1px solid #dfe1e6;
+    border-radius: 32px;
+} */
+.action {
+  width: 327px;
+  height: 48px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.normal_button {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: calc (50% - 40px / 2 + 0.5px);
+  background: #fff;
+  border-radius: 32px;
+}
+
+.recent_works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  gap: 58px;
+  position: absolute;
+  width: 379px;
+  height: 3283px;
+  left: 0;
+  top: 708px;
+}
```

Comment:
- [ ] As stated the project requirement In order to lay out the elements in "Works" section you should use CSS Grid and Flex-box combined (it is mandatory that you use both). **Use CSS Grid in the main container that contains all of the cards and use Flex-box to lay out an individual card.** So Kindly Fix it.